runtime/vm/JVMImage.cpp

```diff
@@ -93,6 +94,9 @@ JVMImage::allocateImageMemory(UDATA size)
 {
 	PORT_ACCESS_FROM_JAVAVM(_vm);
 
+	// Make sure the size is 8 byte aligned, cause that is a requirement for the omr heap (round up)
+	size = ALIGNMENT_ROUND_UP(size);
```

Comment:
might be unnecessary, since allocateImageMem is called only once and we pass it in INITIAL_IMAGE_SIZE which is 8 aligned